### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by yasinabdmahmood**
I followed all the instructions to set up a professional repo according to Microverse standards.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+node_modules/
+.stylelintrc.json
+.hintrc
```
Comment (zieeco) [inline .gitignore]: - Please, kindly remove the `.stylelintrc.json` and `.hintrc` from the `.gitignore` file as these files needs to be visible in your repo.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+* css
+* linter
+
+## Getting started 
+this example shows you how to set up a repository in a professional manner 
+and it will be used through out all the Microverse program .
+
+## Authors
+* GitHub: [yasinabdmahmood](https://github.com/yasinabdmahmood)
+* Twitter: [@YASEEN30093859](https://twitter.com/yasenabd7)
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, kindly update and add the following missing headers from the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md)

1. Getting started: In this header, you should add some instructions on how a user can have your repo locally on their machine. (hot to clone the repo, how to move into the cloned repo), 
2. Prerequisite: e.g `vscode editor`, a `web browser(chrome, etc)`.
3. Contribution header.
4. Show your support header.
5. Acknowledgments.
6. License.
:point_right: The arrangement should follow that use in the [microverse readme tamplate](https://github.com/microverseinc/readme-template/blob/master/README.md) to keep consistency to make your repo appear professional.

*LINTER*
- As it's a requirement for all projects in microverse, please kindly sure your repo has no linter error.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi, @yasinabdmahmood, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Great start :+1: 
- Descriptive PR :+1: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by koachcode — APPROVED** [review]
Hi @yasinabdmahmood , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Status: Approved :green_circle:

Your project is complete. There is nothing else to say about that. :+1:
Go ahead and merge it :muscle:

## [Optional Changes]
### 1. 
title

- [ ] Your PR has a title that is not so descriptive enough. Because **My first pull request** is not describing what are you doing in the project. PR title fully describes what is happening in this branch. You could use **Portfolio: setup and mobile version skeleton** for your PR title. :+1:

### 2
#### Your Summary
branch-name

#### Descriptive Summary Example :heavy_check_mark:
clinton summary



- [ ] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking:

### 3.
- [ ] Your branch name is not descriptive enough. You are using **feature-branch**, you can use  **feature/portfolio-mobile-structure** :+1:

### Once you implement all the changes you should be ready to submit your project for a review :rocket: :rocket:

Cheers and Happy coding!👏👏👏

### Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. You can reach out to me on slack by searching my name (Hammas Bin Saleem) for any explanation.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**